app/backend/src/clj/teet/admin/admin_commands.clj

```diff
@@ -0,0 +1,20 @@
+(ns teet.admin.admin-commands
+  (:require [teet.db-api.core :as db-api]
+            [datomic.client.api :as d]
+            [teet.user.user-roles :as user-roles]
+            [taoensso.timbre :as log]))
+
+(defn- new-user []
+  {:user/id (java.util.UUID/randomUUID)
```

Comment:
That's some `teet.util/common/somesuch` material right there. Gotta have the `uuid` function. :)